types/block.go

```diff
@@ -92,6 +92,26 @@ type (
 		// transactions have ever been committed (since TxnCounter
 		// started being supported).
 		TxnCounter uint64 `codec:"tc"`
+
+		// CompactCert tracks the state of compact certs, potentially
+		// for multiple types of certs.
+		//msgp:sort protocol.CompactCertType protocol.SortCompactCertType
+		//CompactCert map[protocol.CompactCertType]CompactCertState `codec:"cc"`
```

Comment:
nits: should we remove this comment as well?

Reply:
Yep, I'll push this change as well